neofetch

```diff
@@ -317,7 +317,7 @@ get_uptime() {
                     boot="${boot/,*}"
 
                     # Get current date in seconds
-                    now="$(date +%s)"
+                    now="$(printf "%(%s)T")"
```

Comment:
NVM

> http://stackoverflow.com/a/26804450

> Since Bash 4.2, printf has a new modifier %(dateformat)T that, when used with argument -1 outputs the current date with format given by dateformat, handled by strftime(3) (man 3 strftime for informations about the formats).